base/abstractarray.jl

```diff
@@ -683,6 +683,25 @@ copymutable(itr) = collect(itr)
 
 zero{T}(x::AbstractArray{T}) = fill!(similar(x), zero(T))
 
+"""
+    zero!(x::AbstractArray)
+
+Sets all elements of array `x` to 0. Equivalent to `fill!(x, 0)`.
```

Comment:
Typically imperative grammar is used when describing how functions act. So in this case it would be "Set all elements..." rather than "Set**s** all elements..."